nipoppy_cli/nipoppy/config/pipeline.py

```diff
@@ -106,6 +107,36 @@ def get_descriptor_file(self, step_name: Optional[str] = None) -> Path | None:
         """
         return self.get_step_config(step_name).DESCRIPTOR_FILE
 
+
+class BidsPipelineConfig(BasePipelineConfig):
+    """Schema for BIDS pipeline configuration."""
+
+    model_config = ConfigDict(extra="forbid")
+
+    def get_update_doughnut(self, step_name: Optional[str] = None) -> Path | None:
```

Comment:
`nipoppy run` doesn't update the doughnut, `nipoppy bidsify` does

Reply:
Ahh I see - i think it's a bit confusing to have two verbs `get` and `update` in the function name.. because each pipeline run would also look at the doughnut right?